pages/posting/semua.vue

```diff
@@ -165,8 +160,30 @@
     loadingData.value = false
   }
 
+  async function fetchCategory() {
+    const { data } = await $jSiteApi.category.getCategory(
+      siteStore.siteId ?? '',
+      { query: params },
+      { server: false },
+    )
+
+    const categories = data.value?.data as ICategoryData[]
+
+    categories.forEach((cat) => {
+      cat.selected = false
+    })
+
+    filterProps.categories = toRaw(categories ?? [])
```

Comment:
hello @doohanas please handle the fetch error because the data might be null
```suggestion
    const { data, status, error } = await $jSiteApi.category.getCategory(
      siteStore.siteId ?? '',
      { query: params },
      { server: false },
    )
   
   if (status.value === 'success') {
    const categories = data.value?.data as ICategoryData[]

    categories.forEach((cat) => {
      cat.selected = false
    })
    }

    filterProps.categories = toRaw(categories ?? [])
```